plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/ChangedItems.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.git.client.compare;
+
+import org.eclipse.che.ide.api.resources.Project;
+import org.eclipse.che.ide.ext.git.client.compare.FileStatus.Status;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.eclipse.che.ide.ext.git.client.compare.FileStatus.defineStatus;
+
+/**
+ * Describes changed files in git comparison process.
+ *
+ * @author Mykola Morhun
+ */
+public class ChangedItems {
```

Comment:
I'm not sure I understand why do we use term Item here if we're talking exactly about files only according to javadoc? What do you think about something like `AlteredFiles`? In general I think we can't just name this class `ChangedItems` as it does not reflect it's nature, personally I would split it into several classes something like `ChangedItemParser`, `ChangedItemHolder` and `ChangedItem` itself, what do you think?